app/javascript/widget/components/UnreadMessage.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <div class="chat-bubble-wrap">
-    <div class="chat-bubble agent">
+    <div
+      :class="{ campaign: checkMessageIsCampaign }"
```

Comment:
I don't think we need to have a conditional logic here, we can enable click on both campaign messages and normal messages.

Also please consider using <button> instead of <div />